.github/workflows/pr_enforce_labels.yml

```
@@ -10,6 +10,7 @@ permissions:
 
 jobs:
   check-label:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)